modules/ui/modes.js

```diff
@@ -76,6 +77,27 @@ export function uiModes(context) {
 
 
         var debouncedUpdate = _debounce(update, 500, { leading: true, trailing: true });
+        var debouncedClick = _debounce(clickModeButton, 300);
```

Comment:
@quincylvania Not sure if a debounced function is the best way to support a click and a dblclick on an element, but it works for my click rate.